tests/migrations/test_commands.py

```diff
@@ -1352,6 +1352,16 @@ def cmd(migration_count, migration_name, *args):
             self.assertIn("dependencies=[\n('migrations','0001_%s'),\n]" % migration_name_0001, content)
             self.assertIn("operations=[\n]", content)
 
+    def test_makemigrations_with_invalid_custom_name(self):
+        """
+        makemigrations --name should raise CommandError exception if
+        the custom name contains any characters other than alphabet, number and underscore.
```

Comment:
Avoid using conditional wording in test docstring, also use the identifier wording.